front/src/common/SpeedLimitByTagSelector/useStoreDataForSpeedLimitByTagSelector.ts

```diff
@@ -30,10 +31,11 @@ export const useStoreDataForSpeedLimitByTagSelector = ({
   };
   const { data: speedLimitsTagsByInfraId = [], error } =
     osrdEditoastApi.endpoints.getInfraByInfraIdSpeedLimitTags.useQuery(
-      {
-        infraId: infraID!,
-      },
-      { skip: !infraID || isStdcm }
+      infraID
```

Comment:
We have a hard coded list for stdcm we should leave this condition.
```suggestion
      infraID && !isStdcm
```